src/FC/GPS.cpp

```diff
@@ -0,0 +1,84 @@
+#include "GPS.h"
+#include <Wire.h>
+
+
+namespace GPS {
+
+    SFE_UBLOX_GNSS myGNSS;
+    uint32_t gpsUpdatePeriod = 40 * 1000;
+
+    Comms::Packet latLongPacket = {.id = 11};
+    double latitude;
+    double longitude;
+
+
+    Comms::Packet auxPacket = {.id = 12};
```

Comment:
Packet 12 is for something else rn, can you add a line to the E-1 Design spreadsheet for a new packet? Maybe ID 7, since thats unused right now